nemo_skills/evaluation/evaluator.py

```diff
@@ -433,6 +433,7 @@ def eval_lean4(cfg):
     'answer_judgement': dummy_eval,
     'lean4': eval_lean4,
     'mmlu-pro': eval_mmlu_pro,
+    'gpqa': eval_mmlu_pro,
```

Comment:
if it's the same, let's just use mmlu-pro in the default args? Maybe rename it to something more generic, I guess if it's reusable?